templates/show_params.mako

```diff
@@ -271,6 +271,37 @@ ${ job.command_line | h }</pre>
     %endfor
 %endif
 
+%if trans.user_is_admin():
+<h3>Destination Parameters</h3>
+    <table class="tabletip">
+        <thead>
+        <tr>
+            <th>Key</th>
+            <th>Value</th>
+        </tr>
+        </thead>
+        <tbody>
+            <tr><td>Runner</td><td>${ job.job_runner_name }</td></tr>
```

Comment:
What about using `<th>` for `Runner`, `Runner Job ID`, ... and drop `Key`, `Value` above?

Reply:
Yeah that could work!

Reply:
Thanks for considering, I think I didn't make myself very clear, I've pushed a commit to your branch, which is probably easier. Feel free to remove it if you don't like it.

Reply:
Oh, interesting, I had never seen that construct before. Looks fine to me!